recipes/zoltan/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "zoltan" %}
+{% set version = "3.83" %}
+{% set build = 0 %}
+
+{% set mpi = mpi or 'mpich' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://www.cs.sandia.gov/~kddevin/Zoltan_Distributions/zoltan_distrib_v{{ version }}.tar.gz
+  sha256: effceda207d9a25a185296cbbf5fd058864813dc7c8917b18c75b78189e941c7
+
+build:
+  number: {{ build }}
+  skip: true  # [win]
+  {% set mpi_prefix = "mpi_" + mpi %}
+  string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}"
+  run_exports:
+    - {{ name }} * {{ mpi_prefix }}_*
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+  host:
+    - {{ mpi }}
```

it seems you also need ``perl``
```suggestion
    - {{ mpi }}
    - perl
```

---

Isn't `perl` a build dependency, rather than a `host` dependency?

---

I think in this specific case it will not matter much. But, IMHO it should stay in the ``host`` requirements according to 
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host

---

If ``perl`` generates different outputs for different platforms then I agree that should be in ``build``